README.md

```diff
@@ -350,13 +350,13 @@ After this, re-build and restart using `yarn build` and `yarn start` as document
 
 When implementing user interfaces you will need to refer to the [Sketch](https://www.sketchapp.com/)-based mocks that are hosted on InVision: https://mozilla.invisionapp.com/share/YCCYEOFRH#/screens.
 
-If you need to explore the Sketch source file, you will need a license to run Sketch and you also need to install some fonts (which are free). Install [Fira Sans](https://www.fontsquirrel.com/fonts/fira-sans), [Open Sans](https://www.fontsquirrel.com/fonts/open-sans), [Fira Mono](https://www.fontsquirrel.com/fonts/fira-mono) and [Chivo](https://www.fontsquirrel.com/fonts/chivo).
+If you need to explore the Sketch source file, you will need a license to run Sketch and you also need to install some fonts (which are free). Install [Inter](https://www.fontsquirrel.com/fonts/inter) and [Chivo](https://www.fontsquirrel.com/fonts/chivo).
 
 On MacOS, you can use the Homebrew tap [Caskroom-fonts](https://github.com/Homebrew/homebrew-cask-fonts):
 
 ```
 brew tap homebrew/cask-fonts
-brew cask install font-fira-sans font-open-sans font-chivo font-fira-mono
+brew cask install font-inter font-chivo
```

Comment:
What's chivo again?

Reply:
heh, no idea, I just removed the two I knew about.  Remove 'chivo' too?

Reply:
Although maybe this entire section is obsolete - do we still bundle old sketch files?

Reply:
I don't think it's been updated in ages, and I don't know if UX even uses it anymore for mockups. I think we can remove this section, but maybe check with @verdi.

Reply:
> do we still bundle old sketch files?

I don't think so, we removed them from the repo a while ago and I don't know where they are 😅

Reply:
It'd be interesting to know from UX what the situation is with invision but I'm not sure it's a relevant link any way - we were done implementing the site mocks a long time ago so now the reference for the frontend should "look" like it the frontend on prod 😄

Reply:
I agree, I think we can remove the whole section.

Reply:
dropped the section in https://github.com/mozilla/addons-frontend/pull/10110/commits/f3bbaf86a964b1601a577758523e2c978afdc8d6